kafka-cron/consumer/main_test.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/Shopify/sarama"
+	"github.com/Shopify/sarama/mocks"
+	"github.com/berkeli/kafka-cron/types"
+	"github.com/berkeli/kafka-cron/utils"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ExecuteCommand(t *testing.T) {
+	t.Run("should execute hello world", func(t *testing.T) {
+		want := "hello world\n"
+
+		got, err := executeCommand("echo hello world")
+
+		require.NoError(t, err)
+
+		require.Equal(t, want, got)
+	})
+
+	t.Run("should throw error with invalid command", func(t *testing.T) {
+		got, err := executeCommand("invalid command")
+
+		require.Error(t, err)
+
+		require.ErrorContains(t, err, "exit status 127")
+
+		require.Equal(t, "", got)
+	})
+}
+
+func Test_ProcessCommand(t *testing.T) {
+	t.Run("should process command", func(t *testing.T) {
+		producer := mocks.NewSyncProducer(t, nil)
+		cmd := types.Command{
+			Clusters:    []string{"cluster-a"},
+			Description: "hello world",
+			Command:     "echo hello world",
+			MaxRetries:  3,
+			Schedule:    "*/1 * * * *",
+		}
+
+		wg := &sync.WaitGroup{}
+		wg.Add(1)
+
+		processCommand(producer, cmd, wg)
```

Comment:
This test seems to be missing some kind of assertion?

(Also applies to your producer tests)